tests/e2e/specs/LicenseCode.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+
+    'Check if the main p tag with the class license-text is present': function(browser) {
+        browser
+            .init()
+            .assert.elementPresent('.control-label > span')
+            .click('.control-label > span')
+            .assert.elementPresent('.pagination-next')
+            .click('.pagination-next')
+            .assert.elementPresent('.select > select')
+            .click('.select > select')
+            .click('option[value="CC BY-SA 4.0"]')
+            .assert.elementPresent('.pagination-next')
+            .click('.pagination-next')
+            .assert.elementPresent('p[class="license-text"]')
+    },
+    'Check if the a tag in the "Rich Text" redirects to https://creativecommons.org/licenses/by-sa/4.0/?ref=ccchooser': function(browser) {
```

Comment:
I think that a better name for this test would be something like "Check if the link to license code redirects properly".

Reply:
Done